boards/nxp/frdm_mcxn947/frdm_mcxn947_mcxn947_cpu0.yaml

```diff
@@ -27,4 +27,5 @@ supported:
   - regulator
   - adc
   - usb_device
+  - lpmtr
```

Comment:
This yaml definition won't trigger any CI tests, as none have dependency on an IP name like `lptmr`. The tag you need to get counter tests to run is `counter`, which is already present